colossalai/inference/server/README.md

```diff
@@ -1,27 +1,33 @@
 # Online Service
-Colossal-Inference supports fast-api based online service. Simple completion and chat are both supported. Follow the commands below and
-you can simply construct a server with both completion and chat functionalities. For now we only support `Llama` model, we will fullfill
-the blank quickly.
+Colossal-Inference supports fast-api based online service. Simple completion and chat are both supported. Follow the commands below and you can simply construct a server with both completion and chat functionalities. For now we support `Llama`,`Glide_Llama` and `baichuan` model, we will fullfill the blank quickly.
```

Comment:
When describing model names, there exist architectures (LlamaForCausalLM), model category (Llama, or more specifically, Llama, Llama2 and Llama 3), and model cards (meta-llama/Llama-2-7b). You could describes model categories here, such as "Llama 2, Llama 3, Baichuan 2, etc", and skip `Glide_Llama` as it's not a commonly used model structure.

Reply:
fixed